.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.8'
+services:
+  # Update this to the name of the service you want to work with in your docker-compose.yml file
+  actual-development:
+    # Uncomment if you want to override the service's Dockerfile to one in the .devcontainer 
+    # folder. Note that the path of the Dockerfile and context is relative to the *primary* 
+    # docker-compose.yml file (the first in the devcontainer.json "dockerComposeFile"
+    # array). The sample below assumes your primary file is in the root of your project.
+    #
+    # build:
+    #   context: .
+    #   dockerfile: .devcontainer/Dockerfile
+
+    volumes:
+      # Update this to wherever you want VS Code to mount the folder of your project
+      - ..:/workspaces:cached
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: /bin/sh -c "while sleep 1000; do :; done"
```

Comment:
Why is this necessary? Shouldn’t the server keep running?

Reply:
This is some (default) devcontainer magic as the _default_ command is not how you'll interact with the container.
`yarn install` happens as a `postCreateCommand` (to be more transparent to the user)

I considered also starting the server there, but that would be against best practices. (hence "starting it in your preferred way") -- potential room for improvement there